internal/kubernetes/operator/project/project_test.go

```diff
@@ -821,8 +820,8 @@ func Test_buildEncryptionAtREST(t *testing.T) {
 			AzureKeyVault: atlasV1.AzureKeyVault{},
 			GoogleCloudKms: atlasV1.GoogleCloudKms{
 				Enabled:              data.GoogleCloudKms.Enabled,
-				ServiceAccountKey:    data.GoogleCloudKms.ServiceAccountKey,
-				KeyVersionResourceID: data.GoogleCloudKms.KeyVersionResourceID,
+				ServiceAccountKey:    data.GoogleCloudKms.GetServiceAccountKey(),
```

Comment:
There are some conversations regarding dropping all the getters (as they seem redundant), so I don't know if we should use them. Maybe this might be the time to get the team's thoughts on Golang getters for our models?

cc @wtrocki

Reply:
I can create a separate story to remove all the getters.

Reply:
Please do, I'll add it as an agenda topic for Tuesday (we can cancel the story if we elect to keep them)